fast-af/fast-checkout-woocommerce
# Replace the admin menu icon with the “F” favicon

# Description

See #13 

This replaces the dashicon with the Fast logo "F" icon for the WordPress admin menu.

## Default state
![fast-icon-default](https://user-images.githubusercontent.com/636415/112739539-b4da1b80-8f3a-11eb-8bc0-2546f5cf7aef.png)

## Hover state

![fast-icon-hover](https://user-images.githubusercontent.com/636415/112739535-b3a8ee80-8f3a-11eb-9387-20b522f89403.png)

## Active state

![fast-icon-active](https://user-images.githubusercontent.com/636415/112739540-b4da1b80-8f3a-11eb-9a9e-af03e90b2a20.png)

# Testing instructions

Apply the patch and verify that the icon is changed in the WordPress admin menu.

# Checks
